Sources/CALayer+ContentsGravity.swift

```diff
@@ -42,15 +39,15 @@ struct ContentsGravityTransformation {
             return (bounds.height - scaledContents.height) * (1 - layer.anchorPoint.y)
         }
 
-        switch layer.contentsGravityEnum {
+        switch layer.contentsGravity {
```

Comment:
The name of this was `contentsGravityEnum`? I thought this is an internal type for us

Reply:
it was, but now UIKit is using an API compatible with ours, so we can remove our `...Enum` implementation